modules/compliance-custom-node-pools.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: PROCEDURE
+[id="compliance-custom-node-pools"]
+= Creating custom node pools
+
+The Compliance Operator does not maintain a copy of each node pool configuration. The Compliance Operator aggregates consistent configuration options for all nodes within a single node pool into one copy of the configuration file. The Compliance Operator then uses the configuration file for a particular node pool to evaluate rules against nodes within that pool.
+
+If your cluster uses custom node pools outside the default `worker` and `master` node pools, you will need to supply additional variables to ensure the Compliance Operator aggregates a configuration file for that node pool.
```

Comment:
Not sure if this could be a note or could be left as is.

Reply:
I think I'm going to leave it as-is since we are describing the process. Might revisit that later prior to merge, though. Good looking out!